documentation/docs/tirith-installation/developer-mode-installation.md

```diff
@@ -0,0 +1,61 @@
+---
+id: developer-mode-installation
+title: Installation for Developers (Using Docker and Dev Containers in VS Code)
+sidebar_label: Developer Mode Setup
+description: This documentation overviews you about the introduction of the Tirith software installation on your respective operating system.
+keywords:
+  - Tirith
+  - StackGuardian
+# url: https://www.tirith.com/support/docs/getting-started-with-tirith
+site_name: Tirith
+slug: developer-mode-installation/
+---
+
+<script type="application/ld+json"
+  dangerouslySetInnerHTML={{ __html: JSON.stringify({
+   "@context": "https://schema.org",
+    "@type": "BreadcrumbList",
+    "itemListElement": [{
+      "@type": "ListItem",
+      "position": 1,
+      "name": "Home",
+      "item": "https://www.lambdatest.com"
+    },{
+      "@type": "ListItem",
+      "position": 2,
+      "name": "Support",
+      "item": "https://www.lambdatest.com/support/docs/"
+    },{
+      "@type": "ListItem",
+      "position": 3,
+      "name": "Installation",
+      "item": "https://www.lambdatest.com/support/docs/developer-mode-installation/"
```

Comment:
I think we can remove this, also the same for the other md files

Reply:
@refeed i can remove this as of now but later would have to add these again once the deployment will be done

Reply:
alright, then let's keep it